specification/elasticsan/resource-manager/Microsoft.ElasticSan/preview/2024-07-01-preview/elasticsan.json

```diff
@@ -471,6 +471,21 @@
           },
           {
             "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "x-ms-access-soft-deleted-resources",
```

Comment:
With this parameter , does it mean only soft deleted items are returned, or all objects include soft deleted ones are returned? 
Please update description to include this.

Reply:
Only soft deleted items will be returned.